airflow/example_dags/example_bash_decorator.py

```diff
@@ -99,9 +99,13 @@ def _get_files_in_cwd() -> list[str]:
 
     @task.bash
     def get_file_stats() -> str:
+        from shlex import quote
+
         files = _get_files_in_cwd()
         cmd = "stat "
-        cmd += " ".join(files)
+
+        # sdanitizing the file names
```

Comment:
```suggestion
        # Sanitizing the file names
```

By the way, Python 3.8 added `shlex.join` that makes this even easier

```python
shlex.join([stat, *files])
```

Reply:
Oh, didn't know that! Thanks!

Reply:
Yes, works as expected. Pushed a change